lib/nb.lua

```diff
@@ -1,5 +1,13 @@
-local mydir = debug.getinfo(1).source:match("@?" .. _path.code .. "(.*/)")
+local mydir
+if norns then
+  mydir = debug.getinfo(1).source:match("@?" .. _path.code .. "(.*/)")
+elseif seamstress then
+  mydir = debug.getinfo(1).source:match("@?" .. seamstress.state.path .. "/(.*/)")
+end
 local player_lib = include(mydir .. "player")
+if seamstress and _menu == nil then
+  _menu = paramsMenu
+end
```

Comment:
Rylee just fixed this tidbit w/ https://github.com/ryleelyman/seamstress/commit/c1b241dd0443a1cedeb1d9a3192d85a724fbdd31

Reply:
welp, got released fast